logger.go

```diff
@@ -72,7 +72,7 @@ func New(core zapcore.Core, options ...Option) *Logger {
 		core:        core,
 		errorOutput: zapcore.Lock(os.Stderr),
 		addStack:    zapcore.FatalLevel + 1,
-		clock:       systemClock{},
+		clock:       _systemClock,
```

Comment:
AFAIK there will be no allocation anyway

Reply:
Converting the `systemClock{}` struct to the `Clock` interface will cause an allocation.

We can verify this with escape analysis of the old version:

```
$ go build -gcflags=-m 2>&1 | grep logger.go
[...]
./logger.go:75:3: systemClock literal escapes to heap
[...]
```

The new version doesn't have that.